backend/src/services/v2/smtp/smtp.ts

```diff
@@ -86,6 +88,88 @@ export async function requestReviewForAccessRequest(
   await Promise.all(sendEmailResponses)
 }
 
+export async function requestResponsePostReleaseReviewResponse(review: ReviewDoc, release: ReleaseDoc) {
+  if (!config.smtp.enabled) {
+    log.info('Not sending email due to SMTP disabled')
+    return
+  }
+  const reviewResponse = review.responses[0]
+
+  if (!reviewResponse) {
+    log.info('response not found')
+    return
+  }
+  const emailContent = buildEmail(
+    `Release ${release.semver} has been reviewed by ${reviewResponse?.user}`,
+    [
+      { title: 'Model ID', data: release.modelId },
+      { title: 'Reviewed on', data: review.createdAt.toDateString() },
```

Comment:
I don't think that date is when the person responded to the review but when the review was created. I wouldn't worry too much about included that info here anyway as the email should get sent pretty soon after someone responds to the review so the email timestamp could be used.